volatility3/framework/plugins/windows/mftscan.py

```diff
@@ -178,7 +201,7 @@ def parse_mft_records(
             fn_object = symbol_table_name + constants.BANG + "FILE_NAME_ENTRY"
 
             attr_data = attr.Attr_Data.cast(fn_object)
-            file_name = attr_data.get_full_name()
+            file_name = str(attr_data.get_full_name())
```

Comment:
I generally try to get people to leave string objects as strings (in case callers need to get back to that information), but sadly for memory reasons it's probably best to allow it here...  5:S  I guess we can't just do this on the generator so they go out of the plugin as strings but are otherwise carried around as the full types?  My concern would be other people calling this (although the type information doesn't imply they come with provenance information so... yeah, probably ok, I'm just trying to figure out how to generally discourage it except in situations of memory exhaustion)